workspaces/multi-source-security-viewer/plugins/multi-source-security-viewer/src/components/PipelineRunList/PipelineRunTableColumns.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { TableCell, TableHead, TableRow, makeStyles } from '@material-ui/core';
+import React from 'react';
+
+type PipelineRunTableColumn = Record<
+  string,
+  { name: string; props?: Record<string, any> }
+>;
+
+const columns: PipelineRunTableColumn = {
+  pipelineRunId: {
+    name: 'Pipeline Run ID',
+  },
+  type: {
+    name: 'Type',
+  },
+  critical: {
+    name: 'Critical',
+  },
+  important: {
+    name: 'Important',
+  },
+  moderate: {
+    name: 'Moderate',
+  },
+  low: {
+    name: 'Low',
+  },
+  sbom: {
+    name: 'SBOM',
+  },
+  actions: {
+    name: 'Actions',
+  },
+};
+
+const useStyles = makeStyles(() => ({
+  columnHeader: {
+    fontWeight: 'bold',
+    color: '#757575',
```

Comment:
I'd be careful about hard coding colors, this will cause issues for those with custom themes.

Reply:
Agreed, updated to a theme value.